reference/release-notes/1.23.0.md

```diff
@@ -0,0 +1,70 @@
+---
+orphan: true
+---
+# 1.23.0
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.23.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Core stack improvements
+
+* The Anbox Cloud Appliance is overhauled to deliver a much simpler and streamlined version. This new version of the appliance is made available using the [snap epoch feature](https://snapcraft.io/docs/snap-epochs). Existing installations of the appliance are not affected but the old snap with `epoch=0` is {ref}`deprecated <sec-appliance-deprecation>`. For all new installations of the appliance, use the `epoch=1` snap. See the [appliance installation tutorial](https://documentation.ubuntu.com/anbox-cloud/en/latest/tutorial/installing-appliance/) for instructions.
+
+* Anbox Cloud now offers a way to connect to remote Android instances securely using the Android Debug Bridge (ADB). You can set up an ADB connection using the dashboard or the CLI. See {ref}`howto-access-instance` for more information.
+
+### Streaming improvements
+
+< List enhancements to streaming stack here. >
```

Comment:
Any streaming specific improvements?

Reply:
I think there has been some work on upscaling.
@adglkh

Reply:
I don't think there is anything for this release to mention here.

Reply:
Also shoudn't we start to merge core and streaming into one part as the separation between becomes softer?

Reply:
Yes, I have been feeling that for quite some releases now. I have tried something new, if it works for this release, I will update the release template.